src/app/project/FeaturesGrid.js

```diff
@@ -69,12 +74,14 @@ define([
                     renderExpando: true
                 },
                 action: 'Action',
-                subType: 'Treatment/Type'
+                subType: 'Treatment/Type',
+                opacity: {
+                    label: 'Opacity',
```

Comment:
I would like to see this column have no header and only take up as much width as the child widget. which would be around 30px for https://github.com/agrc/wri-web/blob/opacity/src/app/project/resources/FeaturesGrid.styl#L16